dockerfile_scripts/torch-tb-profiler-patch.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [ "$TORCH_PROFILER_GIT" ]; then
+  TORCH_PROFILER_DIR=/tmp/kineto
+  TORCH_TB_PROFILER_VER="v0.2.0"
```

Comment:
optional style preference: you could use the same bash splitting as in apex to allow having this version set in dockerfile instead of here (https://github.com/determined-ai/environments/blob/master/dockerfile_scripts/install_apex.sh#L8)